package.json

```diff
@@ -96,7 +96,7 @@
     "popper.js": "^1.14.1"
   },
   "devDependencies": {
-    "autoprefixer": "^6.0.0",
+    "autoprefixer": "^6.7.7",
```

Comment:
Is there a specific reason for updating those dependencies (according to this pull request)?

Reply:
Have reverted these changes